cmp/tab/TabContainerModel.ts

```diff
@@ -87,6 +87,7 @@ export interface TabContainerConfig {
  * Note: Routing is currently enabled for desktop applications only.
  */
 export class TabContainerModel extends HoistModel {
+    declare config: TabContainerConfig;
```

Comment:
would this ever conflict with a runtime `config` property?

Reply:
It would, but fairly gently.  An app-developer could implement a `config` member with whatever type they want and use it in whatever way they wish.  The `modelConfig` for any components associated with that model will infer the `config` member type, which may not be what that app-developer intended.